test/constants.go

```diff
@@ -3,9 +3,9 @@ package test
 const (
 	// LatestStableMattermostVersion is the most recent stable version of
 	// Mattermost.
-	LatestStableMattermostVersion = "6.2.1"
+	LatestStableMattermostVersion = "6.5.0"
```

Comment:
Since we are bumping it, we might as well do it to the latest versions https://github.com/mattermost/mattermost-server/releases

Reply:
You are right. I will update it to the latest one.